app/models/auction_status.rb

```diff
@@ -19,6 +19,10 @@ def expiring?
     available? && auction.ended_at < (Time.current + 12.hours)
   end
 
+  def work_in_progress?
```

Comment:
I think this is great for now, but we could also consider adding a `work_in_progress` enum so that we don't have to do this crazy stuff